Source/Particles/MultiParticleContainer.H

```diff
@@ -22,9 +22,9 @@
 #include "Particles/RigidInjectedParticleContainer.H"
 
 #ifdef WARPX_QED
-#   include "QED/QedChiFunctions.H"
-#   include "QED/BreitWheelerEngineWrapper.H"
-#   include "QED/QuantumSyncEngineWrapper.H"
+#   include "QedChiFunctions.H"
+#   include "BreitWheelerEngineWrapper.H"
+#   include "QuantumSyncEngineWrapper.H"
```

Comment:
```suggestion
#   include "Particles/ElementaryProcess/QEDInternals/QedChiFunctions.H"
#   include "Particles/ElementaryProcess/QEDInternals/BreitWheelerEngineWrapper.H"
#   include "Particles/ElementaryProcess/QEDInternals/QuantumSyncEngineWrapper.H"
```
Now we add the paths to include statements.

Reply:
Ok! I will comply with this policy in the future... I could not commit directly your suggestion because only `QedChiFunctions.H` is in `QEDInternals`, but I have implemented it.